web/Views/Landlord/Profile.cshtml

```diff
@@ -15,6 +15,25 @@
             </div>
         }
     }
+    @{
+        if (TempData["ToastMessage"] != null)

```

Comment:
Might be worth being a bit more explicit rather than "ToastMessage", could call it e.g. "InviteLinkMessage"

Reply:
Yeah ok done.